libs/txnbuild/rebalance_reserve.go

```diff
@@ -20,6 +22,7 @@ func (rebalanceReserve *RebalanceReserve) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for RebalanceReserve initialises the vtxnbuild struct from the corresponding xdr Operation.
```

Comment:
FromXDR for RebalanceReserve initialises the vtxnbuild struct from the corresponding **XDR** Operation.